packages/kit/src/runtime/client/client.js

```diff
@@ -1861,7 +1864,12 @@ export function pushState(url, state) {
 	if (DEV) {
 		try {
 			// use `devalue.stringify` as a convenient way to ensure we exclude values that can't be properly rehydrated, such as custom class instances
-			devalue.stringify(state);
+			devalue.stringify(
+				is_svelte_5
+					? // @ts-expect-error
```

Comment:
can we add a comment after explaining why we need to ignore a typescript error here?

Reply:
```suggestion
					? // @ts-expect-error This package has a dev dependency on Svelte 4, but `svelte.unstate` only exists in Svelte 5
```